t

```diff
@@ -226,6 +226,8 @@ detect_and_run() {
 }
 
 set_project_root() {
+  [ -n "${PROJECT_ROOT}" ] && return
```

Comment:
I think it would be cleaner to use an if statement instead of using `&&` for control flow.